app/tests/spec/views/sign_in.js

```diff
@@ -786,16 +786,31 @@ define(function (require, exports, module) {
       });
     });
 
-    describe('_engageForm', function () {
-      it('logs the engage event', function () {
-        return view.render()
-          .then(function () {
-            view.afterVisible();
-            assert.isFalse(TestHelpers.isEventLogged(metrics, 'flow.signin.engage'));
-            view.$('form').click();
-            assert.isTrue(TestHelpers.isEventLogged(metrics, 'flow.signin.engage'));
-          });
-      });
+    it('logs the engage event', () => {
+      assert.isTrue(TestHelpers.isEventLogged(metrics, 'flow.signin.begin'));
+      assert.isFalse(TestHelpers.isEventLogged(metrics, 'flow.signin.engage'));
+      view.$('form').click();
```

Comment:
In channel I thought you mentioned clicking on the form w/o doing anything else should not count as an engagement?